nservicebus/messaging/claimcheck/index.md

```diff
@@ -10,6 +10,7 @@ related:
  - samples/databus/file-share-databus
  - samples/databus/custom-serializer
  - samples/databus/blob-storage-databus
+ - samples/databus/redis
```

Comment:
Are we saying that the solution the sample shows can be used in Production and we would support it?

Reply:
That we would support what exactly? If the code as is doesn't work on the NSB side, of course we would support that. Any Redis issues are out-of-scope. The code doesn't use anything we don't already support

Reply:
I think this should be moved into a future-folder

Reply:
Do you mean the sample? Why? The .NET Aspire one isn't/